docs/05_operations/02_commands/01_server-commands.md

```diff
@@ -544,55 +544,73 @@ If you run `remap` (which modifies the  dictionary) after `MigrateDictionary` an
+547644  ALPHA_DATASERVER              11000       HEALTHY        34.20     3.90      ;Lmdb currently uses 1% of available space (total size 2GB)
+547771  ALPHA_EVALUATOR               11003       STANDBY        25.50     2.80
+547686  ALPHA_EVENT_HANDLER           11001       RUNNING        47.90     4.10
 
+```
 
-PID     Process Name                  Port        Status         CPU       Memory
+By default, the command shows the following columns of information:
 
-==================================================================================
 
-9480    AUTH_CONSOLIDATOR             8006        RUNNING        0.30      1.60
+| Column       | Description                                                                                                                                 |
+|--------------|---------------------------------------------------------------------------------------------------------------------------------------------|
+| PID          | The process ID, which is generated by the system                                                                                            |
+| Process Name | The process name                                                                                                                            |
+| Port         | The port where the process is operating; you can change the defaults in the **process.xml** file                                            |
+| Status       | The status of the process (see below)                                                                                                       |
+| CPU          | The percentage of CPU being used by the process. For multi-core CPUs or multiple CPUs, the total percentage of all processes can exceed 100 |
+| Memory       | The percentage of RAM being used by the process                                                                                             |
+| Message      | Displays any relevant message generated by the process                                                                                      |
 
-9392    AUTH_DATASERVER               8002        RUNNING        0.30      1.70
+The key column in the display is **Status**, which can have the following values:
 
-9359    AUTH_MANAGER                  8001        RUNNING        0.30      1.60
+- **RUNNING** the process has just started and is still initialising.
+- **HEALTHY** the process is fully available.
+- **STANDBY** the process is in standby mode; it will take it over if the node becomes Primary
+- **MISSING** there has been an error and the process has not started correctly.
 
-9419    AUTH_PERMS                    8003        RUNNING        0.30      1.80
-```
+- :::info
+If you are using a development environment and you are running a local build via the IntelliJ plugin, you cannot run the `mon` command. The plugin provides a **Mon** tab where you can monitor the status of each process and start and stop each one.
```

Comment:
I'm not sure if we need this box here... essentially you should not run any command if you are in the IntelliJ Development environment, so I think it might be superflous to add this for every command? Maybe something at the top of the page as a warning?

We are in the server commands page anyway.

Reply:
leaving in pace - as per chat